### ISSUE TITLE
Solaris update

### ISSUE THREAD

**Issue by gaborcsardi** [r-hub/rhub#466]
From @brodieg:

> One thing that just came up for me (lucky me), is that CRAN switched their
"patched" machines to "prerelease" during the release window.  Or at least,
they have now.  So e.g. this is what the Solaris ODS machine[1] is running
at the moment:
>
>    using R version 4.1.0 beta (2021-05-08 r80272)
    using platform: i386-pc-solaris2.10 (32-bit)
    using session charset: UTF-8
    using option ‘--no-stop-on-test-error’
>
>Any chance Rhub can update to the same?

**Comment by brodieG** [comment]
> Oh, I guess you don't know because CRAN only uses one of them. If it an encoding issue, then there is a good chance that both fail, so let me update gcc first.

Yes, indeed.  I would expect the problem to manifest on both gcc and ODS if it is R-beta vs. R-patched differences, which is what I think it is.

If that doesn't do it, I would probably wait until the release is done to avoid an extra annoying ODS update cycle for just a week or so before 4.1.

**Comment by brodieG** [comment]
This is great, I can reproduce the failure.  Thanks!
